schema_based_on_diagram.sql

```diff
@@ -0,0 +1,63 @@
+/* Creating tables for clinic database and primery keys*/
+
+create table patients(
+    id int generated always as identity,
+    name varchar(60),
+    date_of_birth date,
+    primary key(id)
+)
+
+create table medical_histories(
+    id int generated always as identity,
+    admitted_at timestamp,
+    patient_id int,
```

Comment:
- [ ] Kindly notice that it is best to add the reference to the foreign key in the creation of the table.

```sql
patient_id int reference patients(id),
```